src/resources/locales/fr.json

```diff
@@ -158,12 +158,18 @@
         "shelf": "Étagère"
     },
     "publication": {
+        "audio": {
```

Comment:
TODO: Spanish. Can we just use Google Translate (or equivalent free service) to obtain the labels for audio "Track" and "Duration"?